example/package.json

```diff
@@ -19,8 +19,8 @@
     "base-64": "^1.0.0",
     "react": "^17.0.2",
     "react-native": "^0.67.1",
-    "react-native-bouncy-checkbox": "^2.1.10",
-    "react-native-phyllo-connect": "^0.2.0",
+    "react-native-bouncy-checkbox": "^3.0.2",
+    "react-native-phyllo-connect": "file:../react-native-phyllo-connect-0.3.0.tgz",
```

Comment:
In release code, change the plugin version to the published version

Reply:
yes now we are going publish , we can change it , i have changed